exporter/oci/export.go

```diff
@@ -35,6 +35,7 @@ const (
 	VariantDocker       = "docker"
 	ociTypes            = "oci-mediatypes"
 	keyForceCompression = "force-compression"
+	keyCompressionLevel = "compression-level"
```

Comment:
Should we change force to `compression-force` in follow-up and deprecate old field?